tests/cases/fourslash/renameStringLiteralTypes.ts

```diff
@@ -0,0 +1,18 @@
+/// <reference path='fourslash.ts' />
+
+
+////interface AnimationOptions {
+////    deltaX: number;
+////    deltaY: number;
+////    easing: "ease-in" | "ease-out" | "[|ease-in-out|]";

```

Comment:
yes, the whole string literal is one token we do not break it apart and scan inside it.